### DISCUSSION TITLE
ytdl_hook路径无法读取

### DISCUSSION THREAD

**Discussion by OrangeYoon** [hooke007/mpv_PlayKit#466] [mpv 通用]
![image](https://github.com/hooke007/MPV_lazy/assets/169504647/a2568c21-6de0-4b3d-82c1-6bc29b67e816)
![image](https://github.com/hooke007/MPV_lazy/assets/169504647/73f13802-039a-4e5d-96e4-84faa3f386e1)
![image](https://github.com/hooke007/MPV_lazy/assets/169504647/ab2e8ed7-7e10-4134-82e9-a2dfd43047c3)
![image](https://github.com/hooke007/MPV_lazy/assets/169504647/8d6a54eb-4d03-4b9b-918f-555a320f34ad)
十分的奇怪，砖家们帮我诊断一下，谢谢

**Comment by hooke007** [comment]
你这个是脚本设置文件，喂给mpv.conf 有什么用？
include读取的也只能是mpv.conf
你可以先分清楚各个配置文件都是做什么的

**Reply by OrangeYoon** [reply]
那我这个ytdl_hook.config应该放在哪里，github上是在一个文件夹里，它能读到吗？

**Reply by hooke007** [reply]
很多问题都过于基础和简单，wiki和手册都有写
如果你有想法自己搭配置框架
我建议完全了解mpv配置的工作机制

https://hooke007.github.io/unofficial/mpv_start.html
